src/pykeen/triples/triples_factory.py

```diff
@@ -471,3 +471,43 @@ def new_without_relations(self, relations: Collection[str]) -> 'TriplesFactory':
         logger.info(f'removing {len(relations)}/{self.num_relations} relations'
                     f' and {idx.sum()}/{self.num_triples} triples')
         return TriplesFactory(triples=self.triples[idx])
+
+    def entity_word_cloud(self, top: Optional[int] = None):
+        """Make a word cloud based on the frequency of occurrence of each entity.
```

Comment:
Add information that this relies on being run within a Jupyter notebook?

Reply:
Resolved in 466d0fe8